cmd/csv2parquet/main.go

```diff
@@ -0,0 +1,435 @@
+package main
+
+import (
+	"encoding/csv"
+	"encoding/json"
+	"errors"
+	"flag"
+	"fmt"
+	"io"
+	"log"
+	"os"
+	"sort"
+	"strconv"
+	"strings"
+	"unicode/utf8"
+
+	goparquet "github.com/fraugster/parquet-go"
+	"github.com/fraugster/parquet-go/parquet"
+	"github.com/fraugster/parquet-go/parquetschema"
+)
+
+var printLog = func(string, ...interface{}) {}
+
+func main() {
+	inputFile := flag.String("input", "", "CSV file input")
+	typeHints := flag.String("typehints", "", "type hints to help derive parquet schema. A comma-separated list of type hints in the format <column_name>=<parquettype>; valid parquet types: "+strings.Join(validTypeList(), ", "))
+	outputFile := flag.String("output", "", "output parquet file")
+	rowgroupSize := flag.Int64("rowgroup-size", 0, "row group size in bytes; if value is 0, then the row group size is unbounded")
```

Comment:
can we have a meaningful default value here? It can lead to problems with unlimited row group size

Reply:
I set it to 100 MiB.